_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,986 @@
+splitting and switching_ (PSS). His code allows nodes to split incoming
+payments into [multiple parts][topic multipath payments], which can take
+different routes before reaching the final recipient. For instance, a
+payment from Alice to Bob could be partly routed through Carol. This
+technique significantly hinders balance discovery attacks, where
+attackers probe channel balances to track payments across the network.
+Van Dam's research showed a 62% decrease in information gained by
+attackers using PSS.  Additionally, PSS offers increased Lightning
+Network throughput and may help mitigate [channel jamming attacks][topic
+channel jamming attacks].
+
+{:#sidepools}
+Developer ZmnSCPxj [proposed][oct sidepool] a concept called _sidepools_
+that aims to enhance LN's liquidity management. Sidepools involve
+multiple forwarding nodes depositing funds into a multiparty offchain
+state contract similar to LN channels. This allows funds to be
+redistributed among participants offchain. For instance, if Alice, Bob,
+and Carol each start with 1 BTC, the state can be updated so that Alice
+has 2 BTC, Bob 0 BTC, and Carol 1 BTC.  Participants would still use and
+advertise regular LN channels, and if those channels became imbalanced,
+a rebalancing can be done through an offchain peerswap within the state
+contract. This method is private to participants, requires less onchain
+space, and likely eliminates offchain rebalancing fees, thus improving
+revenue potential for forwarding nodes and reliability for LN payments.
+However, it requires a multiparty state contract, which is untested in
+production.  ZmnSCPxj suggests using [LN-Symmetry][topic eltoo] or
+duplex payment channels as a foundation, with duplex channels being more
+feasible despite their inability to remain open indefinitely.
+
+{:#assumeutxo}
+October saw the [completion][oct assumeutxo] of the first phase of the
+[assumeUTXO project][topic assumeutxo], containing all the remaining
+changes necessary to both use an assumedvalid snapshot chainstate and do
+a full validation sync in the background. It makes UTXO snapshots
+loadable via an RPC.  Although the feature set will not be available on
+mainnet until activated, this merge marked the culmination of a
+multi-year effort. The project, proposed in 2018 and formalized in 2019,
+will significantly improve the user experience of new full nodes first
+coming onto the network.
+
+{:#p2pv2}
+Also [accomplished][oct p2pv2] by the Bitcoin Core project in October was the
+addition of support for [version 2 encrypted P2P transport][topic v2 p2p
+transport] as specified in [BIP324][]. The feature is currently disabled
+by default but can be enabled using the `-v2transport` option.
+Encrypted transport helps improve the privacy of Bitcoin users by
+preventing passive observers (such as ISPs) from directly determining
+which transactions nodes relay to their peers. It’s also possible to use
```

Comment:
```suggestion
Encrypted transport improves the privacy of Bitcoin users by
preventing passive observers (such as ISPs) from observing
which transactions nodes relay to their peers. While previously a man-in-the-middle attacker would have been able to simply observe or censor relay of certain transactions, either reading or modifying traffic would now require an active observer that maintains the full protocol state with both Bitcoin nodes whose traffic is being observed. It’s also possible to use
```

Reply:
I think this is too much detail given that we link to our earlier post.  Also, "maintaining the full protocol state" for BIP324 isn't a big deal.  I don't see how it'd be significantly more complicated than performing a pure TCP MITM.  As an example, there's plenty of tools for MITM'ing SSL connections.